src/coreclr/scripts/superpmi.py

```diff
@@ -719,6 +724,9 @@ def __collect_mc_files__(self):
             root_env["SuperPMIShimLogPath"] = self.temp_location
             root_env["SuperPMIShimPath"] = self.jit_path
 
+            if self.coreclr_args.pmi and self.pmi_path_directories:
+                root_env["PMIPATH"] = ";".join(self.pmi_path_directories)
```

Comment:
How is `pmi_path` passed? I assume it is already `;` separated, right? We don't need `.join()` in that case? Or do we need to convert `pmi_path` to array while setting `pmi_path_directories`?

Reply:
> How is pmi_path passed?
When you run `superpmi.py` you can pass multiple directories with
```
superpmi.py collect <other args> -pmi_path $dir1 -pmi_path $dir2
```

> I assume it is already ; separated, right?

No, internally the values are represented as list `[$dir1, $dir2]`. Hence, the `";".join()`